internal/mdxc/application_identity.go

```diff
@@ -8,14 +8,17 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/types"
-	"google.golang.org/api/iam/v1"
 )
 
 type AWSApplicationIdentityInputData struct {
 	AssumeRolePolicy types.String `tfsdk:"assume_role_policy"`
 }
 type GCPApplicationIdentityInputData struct {
-	Placeholder types.String `tfsdk:"placeholder"`
+	Kubernetes GCPKubernetesIdentityInputData `tfsdk:"kubernetes"`
+}
+type GCPKubernetesIdentityInputData struct {
```

Comment:
FYI, we may pull this out to its own block (instead of nesting in GCP) since all clouds end up needing this for k8s workload identity.  In AWS is wrapped in the `assume_role_policy` and its as-yet unclear what Azure will look like.

Reply:
I had that same thought! I wasn't sure (yet) if AWS / Azure will need specific things for their k8s but hopefully we can generalize.